sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -258,20 +260,21 @@ export class EducationProgramOfferingController extends BaseController {
   @AllowAuthorizedParty(AuthorizedParties.institution)
   @HasLocationAccess("locationId")
   @Get(
-    "location/:locationId/education-program/:programId/program-year/:programYearId/offerings-list",
+    "location/:locationId/education-program/:programId/program-year/:programYearId/selectedOffering/:selectedOffering/offerings-list",
   )
   async getProgramOfferingsForLocationForInstitution(
     @Param("locationId") locationId: number,
     @Param("programId") programId: number,
     @Param("programYearId") programYearId: number,
+    @Param("selectedOffering") selectedOffering: OfferingIntensity,
```

Comment:
are you passing `offering id` or `offering intensity`?. this is confusing. can you use proper name

Reply:
It is offering intensity selected by the student. The aim is to ensure that the list of offerings returned in the drop down match only what the student selected. E.g if the student selected full time, then only full time offerings should be displayed. Same for part time.